src/zeit/content/article/edit/citation.py

```diff
@@ -27,3 +27,16 @@ class Factory(zeit.content.article.edit.block.BlockFactory):
 
     produces = Citation
     title = _('Citation')
+
+
+# Could be defined as an Adapter (IArticle -> ICitation) as well
+# Im not sure, when to use just a function or an Adapter
```

Comment:
Ich glaub da gibt es keine 100%ige Regel. Also, klar, wenn _irgendwas irgendwie_ dispatcht werden soll, Adapter nehmen. Aber bei sowas hier, wo es nur eine Implementierung gibt (und nach allem was wir wissen je geben wird), hat's eher mit Ästhetik zu tun. Ich persönlich würde zu Funktion tendieren, denn ich seh irgendwie noch nicht, dass "IFooBlock(article)" für "gib mir das erste Foo-Modul" so ne dolle Abstraktion ist. (Wobei wir genau sowas in `IArticle.main_image_block` schonmal haben... hmmm. :)